Mehdi_test/mod_2.nit

```diff
@@ -0,0 +1,13 @@
+import date_f
+
+var test  = new Date(2014, 01, 09)
+var test2 = new Date(2016, 01, 09)
+var test3 = new Date(2015, 01, 12)
+var res   = test.date_diff(test2,2)
```

Comment:
I understand the need to align things, but this is not correct: no more than one space before a math operator